.cloudbuild/scripts/internal/github/meta.go

```diff
@@ -0,0 +1,58 @@
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gravitational/trace"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	metadataURL = "https://api.github.com/meta"
```

Comment:
It turns out that there's a ridiculous rate limit of 60 per hour per IP, for unauthenticated requests :(
If we do this twice per PR (for unit and integration), that's 30 per hour per IP, is that going to be enough? This should only be hit when running in -private anyway.

Reply:
Yeah, should only happen in `private`, a the deploy key won't be set for straight `teleport`. I'm curious as to how the requests from multiple builds would appear to GitHub, though.

They should all be from separate VMs that _may_ have different public addresses. Or they may all be NATed behind something. I don't know enough about the build node networks setup to say.

Either way, though, I think the worst case of 30 PRs/hour is probably OK, especially given that an individual PR takes 20mins to do a full test.